### ISSUE TITLE
What are maximum lengths for any sequence?

### ISSUE THREAD

**Issue by vsoch** [vsoch/freegenes#5]
Specifically, these in the parts table:

```python
    # Sequences (what are the maximum lengths here?)
    original_sequence = models.CharField(max_length=250, null=False, required=True)
    optimized_sequence = models.CharField(max_length=250, null=False, required=True)
    synthesized_sequence = models.CharField(max_length=250, null=False, required=True)
    full_sequence = models.CharField(max_length=250, null=False, required=True)
```

**Comment by Koeng101** [comment]
Optimally, no maximum length. Practically, as large as you can make them (with a **BARE MINIMUM** of being ~10,000)

**Comment by vsoch** [comment]
This was addressed in #7, with further discussion in #9 , and will be decided to be stored as TextField (in the database, TOAST compression) or file object based on the intended use case. I've added this as a question to #11 so we can close here.
